app/views/users/new.html.erb

```diff
@@ -1,3 +1,24 @@
 <% provide(:title, (t ".title")) %>
 <h1><%= t ".topic" %></h1>
-<p><%= t ".content" %></p>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+
+      <%= f.label :email %>
+            <%= f.email_field :email, class: "form-control" %>
```

Comment:
indent

Reply:
nếu @user bằng nil thì bên view sẽ báo lỗi này anh. "undefined method `name' for nil:NilClass"